app/src/main/java/com/github/se/cyrcle/ui/list/ListScreen.kt

```diff
@@ -145,21 +133,31 @@ fun FilterHeader(
     selectedCapacities: Set<ParkingCapacity>,
     onAttributeSelected: (ParkingAttribute) -> Unit,
     onlyWithCCTV: Boolean,
-    onCCTVCheckedChange: (Boolean) -> Unit
+    onCCTVCheckedChange: (Boolean) -> Unit,
+    parkingViewModel: ParkingViewModel = viewModel(factory = ParkingViewModel.Factory)
```

Comment:
The code seems to work for the moment but please be carefull when passing a default ViewModel, I think it instanciates a new one and thus have different state than the one made in the Main. It seems ok since you don't actualy use the default value.